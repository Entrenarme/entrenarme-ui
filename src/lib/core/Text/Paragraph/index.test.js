import * as React from 'react';
import { render } from 'react-testing-library';
import 'react-testing-library/cleanup-after-each';

import P from './index';

test('renders "Paragraph" with no props', () => {
  const { container } = render(<P>Paragraph</P>);
  expect(container.firstChild).toMatchSnapshot();
});

test('renders "Paragraph" with l prop', () => {
  const { container } = render(<P l>Paragraph</P>);
  expect(container.firstChild).toMatchSnapshot();
});

test('renders "Paragraph" with s prop', () => {
  const { container } = render(<P s>Paragraph</P>);
  expect(container.firstChild).toMatchSnapshot();
});
