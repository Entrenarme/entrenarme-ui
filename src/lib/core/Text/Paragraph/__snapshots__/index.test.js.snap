// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders "Paragraph" with align center 1`] = `
<p
  class="sc-bdVaJa flMesl"
>
  Paragraph
</p>
`;

exports[`renders "Paragraph" with l prop 1`] = `
<p
  class="sc-bdVaJa gisIDb"
>
  Paragraph
</p>
`;

exports[`renders "Paragraph" with no props 1`] = `
<p
  class="sc-bdVaJa gjUjvw"
>
  Paragraph
</p>
`;

exports[`renders "Paragraph" with s prop 1`] = `
<p
  class="sc-bdVaJa kHjWkw"
>
  Paragraph
</p>
`;
